rules/windows/process_creation/proc_creation_win_webdav_lnk_execution.yml

```diff
@@ -0,0 +1,34 @@
+title: Potentially Suspicious WebDAV LNK Execution
```

Comment:
```suggestion
title: Suspicious WebDAV LNK Execution
```
for a high rule we don't put a "potentially", do we?

Reply:
I put it there and should be medium

Reply:
yeah or that way round